### PR TITLE
VE-1765: Initial port of VisualEditor source dialog from old VE to new VE

### PR DESCRIPTION
**Pull request by inez**
https://wikia-inc.atlassian.net/browse/VE-1765

### CODE REVIEW

**Comment by inez** [comment]
@lizlux Please review whenever you get a chance. (Sooner or later I'm going to merge it anyways but feedback is still appreciated - even if merged.)

File: extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaSourceModeDialog.scss [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+			height: 100%;
+			line-height: 140%;
+			width: 100%;
+		}
+	}
+	.ve-ui-wikiaSourceModeDialog-helplink {
+		font-weight: bold;
+		line-height: 3em;
+	}
+}
```
Comment (lizlux) [inline extensions/VisualEditor/wikia/modules/ve/ui/styles/ve.ui.WikiaSourceModeDialog.scss]: New line at end of file

File: extensions/VisualEditor/wikia/modules/ve/ui/tools/ve.ui.WikiaDialogTool.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+OO.inheritClass( ve.ui.WikiaSourceModeDialogTool, ve.ui.DialogTool );
+
+ve.ui.WikiaSourceModeDialogTool.static.name = 'wikiaSourceMode';
+ve.ui.WikiaSourceModeDialogTool.static.group = 'object';
+ve.ui.WikiaSourceModeDialogTool.static.icon = 'source';
+ve.ui.WikiaSourceModeDialogTool.static.title =
+	OO.ui.deferMsg( 'wikia-visualeditor-dialogbutton-wikiasourcemode-tooltip' );
+ve.ui.WikiaSourceModeDialogTool.static.commandName = 'wikiaSourceMode';
+ve.ui.toolFactory.register( ve.ui.WikiaSourceModeDialogTool );
```
Comment (lizlux) [inline extensions/VisualEditor/wikia/modules/ve/ui/tools/ve.ui.WikiaDialogTool.js]: New line at end of file

**Comment by lizlux** [comment]
Just a couple minor comments. Since this is all existing code that's just getting added back in after a fresh VE install, it should be fine.

**Pull request merged**